dashboard2/pages_list.md

```diff
@@ -0,0 +1,148 @@
+
+*Make sure that you have selected a target language* to gain access to many powerful features in the page list. If you do not (or there is no target language on the project at all), neither the Workbench links, nor the Previews will be available, and most of the context menu options will be greyed out.
+
+## Pages, resources, external links & unknown
+
+Here you can see all the items that the proxy is aware of (either because of crawls or user visits). They are split into 4 categories: pages, resources, external links and unknown. 
+
+Pages are links that contain translatable content. By default, these are the HTML pages on the project but you can move others (like JSONs) from the other sections. 
+
+Resources are images, PDFs and similar that the proxy can't automatically extract content from. However, you may want to localise these by uploading a replacement. 
+
+External links (like those pointing to YouTube videos) can be replaced (e.g. if the video is available in the target language). Note that if you are working on a project for `www.example.com`, `blog.example.com` is considered *external* to it
+
+There should be no entries under Unknown but if you can't find something you may want to check it here. 
+
+Above the list itself, there are multiple actions you can do and filters. These are:
+
+- **Switch to flat/folder view**: Switch between the flat view (default for resources), which is just a list of URLs similar to the old Dashboard and the folder view (default for pages) that allows you to see the pages as if they were files on your computer. External links are only available in flat view so this option is hidden for them.
+ 
+- **Show/hide preview** (only for resources): Whether to load the images or just display their filename. If you have a slow connection or a data cap, we recomment hiding these.
+
+- **Export URL list**: Create a CSV or Excel export of the current list. You can filter the links by translation state or free text search. Due to Excel's limitation on the number of lines per sheet, a CSV export is recommended for large-scale projects.  
+
+- **Add new**: Run a *page-specific crawl*. Use it to add new pages or refresh already extracted pages in a "surgical" fashion. HTTPS URLs are added using fully qualified URLs. If you specify the pathname only, such as `/about-us.html`, the proxy will default to requesting the page over HTTP. The same applies to any resource that is available via both HTTP and HTTPS: add both URLs to ensure that you have all possible versions of the page or resource.
+
+- **Mass page exclusion**: Enter a list of URLs that you wish to exclude. Use this when you can't add an exclusion rule. You also have the option to "un-exclude" aka include some pages again.
+
+- **Update translation progress**: Updates the bars showing the progess. This can take from a few seconds to a few minutes of time to finish, depending on the amount of content to be aggregated.
+
+- **Reload**
```

Comment:
What does this one do? Leaving an item with no definition just looks out of place.